### PR TITLE
1815 read temporal formulas with spec+ temporal rather than next+init+ inv

### PR DESCRIPTION
**Pull request by p-offtermatt**
- [x] Tests added for any new code
- [x] Ran `make fmt-fix` (or had formatting run automatically on all files edited)

[unreleased]: https://github.com/informalsystems/apalache/tree/unstable/.unreleased

### CODE REVIEW

File: docs/src/apalache/features.md [hunk 1]
```diff
@@ -3,12 +3,9 @@ Here is the list of the TLA+ language features that are currently supported by A
...
 
 At the moment, Apalache is able to check state invariants, action invariants,
-trace invariants as well as inductive invariants. (See the [page on
+temporal properties, trace invariants as well as inductive invariants. (See the [page on
 invariants](https://apalache.informal.systems/docs/apalache/principles/invariants.html) in
-the manual.) Which means that you can only check safety properties with
-Apalache, unless you employ a [liveness-to-safety] transformation in your spec.
-In general, we do not support checking liveness properties.  If you would like
-to see liveness implemented, upvote the [liveness feature].
+the manual.) To check liveness/temporal properties, we employ a [liveness-to-safety] transformation.
```
Comment (thpani) [inline docs/src/apalache/features.md:8]: This was carried over, but it needs an extra `[]` to be valid markdown.
```suggestion
the manual.) To check liveness/temporal properties, we employ a [liveness-to-safety][] transformation.
```

File: mod-tool/src/main/scala/at/forsyte/apalache/tla/Tool.scala [hunk 2]
```diff
@@ -244,6 +246,7 @@ object Tool extends LazyLogging {
 
   private def runCheck(executor: PassChainExecutor, check: CheckCmd): Int = {
     setCoreOptions(executor, check)
+    print(executor.options.get[List[String]]("checker", "temporal"))
```
Comment (thpani) [inline mod-tool/src/main/scala/at/forsyte/apalache/tla/Tool.scala:249]: This looks like some left-over debug output?

File: tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/passes/TemporalPassImpl.scala [hunk 3]
```diff
@@ -31,14 +31,15 @@ class TemporalPassImpl @Inject() (
   override def execute(tlaModule: TlaModule): PassResult = {
     logger.info("  > Rewriting temporal operators...")
 
-    val newModule = options.get[List[String]]("checker", "inv") match {
+    val temporalProps = options.get[List[String]]("checker", "temporal")
+    val newModule = temporalProps match {
       case None =>
-        logger.info("  > No invariants specified, nothing to encode")
+        logger.info("  > No formula specified, nothing to encode")
```
Comment (thpani) [inline tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/passes/TemporalPassImpl.scala:37]: "formula" is maybe a bit ambiguous to understand what went wrong, shall we say
```suggestion
        logger.info("  > No temporal property specified, nothing to encode")
```

File: mod-tool/src/main/scala/at/forsyte/apalache/tla/Tool.scala (hunk 2 above)
Comment (p-offtermatt) [reply mod-tool/src/main/scala/at/forsyte/apalache/tla/Tool.scala]: Yes, thanks!

**Comment by p-offtermatt** [comment]
Retired in favor of https://github.com/informalsystems/apalache/pull/1874

**Pull request closed without merging**